src/inventory.cc

```diff
@@ -4755,6 +4756,31 @@ static int _barter_attempt_transaction(Object* a1, Object* a2, Object* a3, Objec
     return 0;
 }
 
+static int _barter_get_quantity_moved_items(
+    Object* item,
+    int maxQuantity,
+    bool fromPlayer,
+    bool fromInventory
+) {
```

Comment:
```suggestion
static int _barter_get_quantity_moved_items(Object* item, int maxQuantity, bool fromPlayer, bool fromInventory)
{
```
Eliminates Clang warning.